developing/learning/radical_candor.md

```diff
@@ -0,0 +1,8 @@
+# [Learning Resources](./README.md) / "Radical Candor"
+
+_Radical Candor_ is basically the Trussel Manager Standard Issue book. It informs a lot of how Trussels aspire to comport themselves with each other.
+
+## Links
+
+* [Radical Candor coaching site](https://www.radicalcandor.com/)
+* [Amazon link](https://www.amazon.com/Radical-Candor-Kim-Scott/dp/B01KTIEFEE)
```

Comment:
Should we point to the revised edition that was just released a few days ago? https://www.amazon.com/Radical-Candor-Revised-Kick-Ass-Humanity-ebook/dp/B07P9LPXPT

Reply:
Done